data/time_series.go

```diff
@@ -526,7 +526,7 @@ func labelsTupleKey(l Labels) (string, error) {
 }
 
 // SortWideFrameFields sorts the order of a wide time series Frame's Fields.
-// It the frame is not a WideFrame, than an error is returned.
+// It the frame is not a WideFrame, then an error is returned.
```

Comment:
```suggestion
// If the frame is not a WideFrame, then an error is returned.
```

Reply:
Hah! Can't believe I breezed right by that one!